resources/views/emails/status_certificate_request.blade.php

```diff
@@ -0,0 +1,7 @@
+@component('mail::message')
+<h1>@lang('mail.dear') {{ $recipient }}!</h1>
+<p>
+@lang('mail.status_cert_request', ['user' => $user]) <a href="{{ $url }}">@lang('mail.show')</a>
```

Comment:
You can use the button component for links (see `approved_registration` email blade).